src/components/TableWrapper.tsx

```diff
@@ -0,0 +1,10 @@
+import { FunctionComponent, ReactNode } from 'react'
+
+interface TableWrapperProps {
+    children: ReactNode
+    className?: string
+}
+
+export const TableWrapper: FunctionComponent<TableWrapperProps> = ({ className = '', children }) => (
```

Comment:
I initially figured it would be helpful for blog writers to have the option to style the table, but I realize now that could potentially introduce inconsistencies. I agree that we should remove this prop: 45f2503!

Reply:
Totally, I was thinking the same. Maybe since we're not passing any special styles currently to tables it'd be good to restrict the variability. Thanks Tim!

Reply:
Totally agree here! Prop drilling can be a nightmare! Lol! Restricting if the prop isn't used would also help to stick with our design consistency ✨